pkg/network/http/ebpf_gotls.go

```diff
@@ -193,33 +267,17 @@ func (p *GoTLSProgram) handleNewBinary(binPath string) {
 // addInspectionResultToMap runs a binary inspection and adds the result to the map that's being read by the probes.
 // It assumed the given path is from /proc dir and gets the pid from the path. It will fail otherwise.
```

Comment:
please modify the documentation as `It assumed the given path is from /proc dir and gets the pid from the path` is incorrect anymore